charts/sscs-tya-notif/values.yaml

```diff
@@ -45,6 +45,9 @@ java:
     MYA_LINK: "https://sscs-cor-frontend-aat.service.core-compute-aat.internal/sign-in?tya=appeal_id"
     HEARING_INFO_LINK: "https://track-appeal.{{ .Values.global.environment }}.platform.hmcts.net/abouthearing/appeal_id"
     CLAIMING_EXPENSES_LINK: "https://track-appeal.{{ .Values.global.environment }}.platform.hmcts.net/expenses/appeal_id"
+    MYA_EVIDENCE_LINK: "https://sscs-cor-frontend-aat.service.core-compute-aat.internal/support-evidence"
```

Comment:
that's pointing to the old AAT URL. The new one is this: sscs-cor.aat.platform.hmcts.net